astroquery/heasarc/core.py

```diff
@@ -1,92 +1,247 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-from typing import Union
+import os
+
+import shutil
+import requests
+import tarfile
 import warnings
-from io import StringIO, BytesIO
+import numpy as np
 from astropy.table import Table
-from astropy.io import fits
 from astropy import coordinates
 from astropy import units as u
-from ..query import BaseQuery
-from ..utils import commons
-from ..utils import async_to_sync
+from astropy.utils.decorators import deprecated, deprecated_renamed_argument
+
+import pyvo
+
+from astroquery import log
+from ..query import BaseQuery, BaseVOQuery
+from ..utils import commons, async_to_sync, parse_coordinates
 from ..exceptions import InvalidQueryError, NoResultsWarning
 from . import conf
 
-__all__ = ['Heasarc', 'HeasarcClass']
```

Comment:
Please don't remove this, we don't want to expose all of the above imports, etc in the public namespace

Reply:
Thanks for spotting this. I think it was removed by mistake.